schemas/router_config_schema.json

```diff
@@ -0,0 +1,8342 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Configuration",
+  "description": "The configuration for the router.\n\nCan be created through `serde::Deserialize` from various formats, or inline in Rust code with `serde_json::json!` and `serde_json::from_value`.",
```

Comment:
This description is a bit unfortunate as it shows up in the editor:

![image](https://github.com/user-attachments/assets/a773f534-f1ab-454e-af2f-2c7f9b500a63)

Reply:
Yeah, this is something we should run by the router team. The point of this schema is to get editor support, so we should be updating our internal docs to be more useful for external consumers